content/_data/changelogs/lts.yml

```diff
@@ -6672,6 +6672,295 @@
+      - timja
+    pr_title: JENKINS-67689 Don't hide cancel button with long build names
+    message: |-
+      Truncate long build names again (regression in 2.332).
+
+  - type: bug
+    category: bug
+    pull: 6264
+    issue: 67753
+    authors:
+      - NotMyFault
+    pr_title: '[JENKINS-67753] Overwrite "grey" balls with modern build status'
+    message: |-
+      Overwrite grey balls icon with the modern "not built" status.
+
+  - type: bug
+    category: regression
+    pull: 6261
+    authors:
+      - dependabot[bot]
+    pr_title: "[JENKINS-67000] Bump remoting from 4.11.2 to 4.12"
+    references:
+      - pull: 5983
+      - issue: 67000
+      - url: https://github.com/jenkinsci/remoting/releases/tag/remoting-4.11.2
+        title: Remoting 4.11.2 changelog
+      - url: https://github.com/jenkinsci/remoting/releases/tag/remoting-4.12
+        title: Remoting 4.12 changelog
+    message: |-
+      Update remoting from 4.11 to 4.12 to allow Java web start agents to connect (regression in 2.318).
+
+  lts_changes: # compared to lts_predecessor 2.319.3 (selected by personal review)
+
+  # Agent to controller subsystem is now always enabled
+
+  - type: major rfe
+    category: major rfe
+    pull: 5885
+    issue: 67173
+    authors:
+      - daniel-beck
+    pr_title: "[JENKINS-67173] Remove Callable allowlist and FilePath agent-to-controller\
+      \ support"
+    message: |-
+      The agent-to-controller security subsystem is now always enabled.
+      The admin-customizable allowlists for callables and file paths have been removed.
+      The ability to access some files on the controller from agents has been removed.
+      Some plugins are incompatible with this change and may need to be updated.
+    references:
+      - pull: 5885
+      - issue: 67173
+      - url: https://issues.jenkins.io/issues/?jql=%22Epic%20Link%22%20%3D%20JENKINS-67173
+        title: the issue tracker
+
+  # Modernise Jenkins components
+
+  - type: major rfe
+    category: major rfe
+    pull: 5707
+    authors:
+      - basil
+    references:
+      - pull: 5707
+      - issue: 36779
+      - url: https://github.com/jenkinsci/jep/tree/master/jep/233
+        title: JEP-233
+      - url: https://guava.dev/
+        title: Guava web site
+      - url: https://github.com/google/guava/releases/tag/v31.0.1
+        title: Guava 31.0.1 changelog
+    pr_title: "[JEP-233] Upgrade Guava from 11.0.1 to latest"
+    message: |-
+      Upgrade the Guava library from 11.0.1 (released on January 9, 2012) to 31.0.1 (released on September 27, 2021).
+      Plugins have already been prepared to support the new version of Guava.
+      Use the Plugin Manager to upgrade all plugins before <u>and</u> after upgrading Jenkins.
+
+  # UI improvements
+
+  - type: rfe
+    category: rfe
+    pull: 5851
+    authors:
+      - janfaracik
+    pr_title: Modernise the table design
+    message: |-
+      Modernise the table design and add support for Ionicons.
```

Comment:
could we collapse all these modernise entries? it looks weird repeating the same thing 5 times?

Reply:
Collapsed the modernisation entries into a single entry in 800e38fa1dd95698ba3d43a42ba49bad3ce684b1.  

I left a few items outside of that single entry because it didn't feel as comfortable to mention "upload plugin from a URL" in the same entry as "modernise tables".  Let me know if you'd prefer a different approach.